wp-content/themes/launchframe/assets/src/scss/_custom_variables.scss

```diff
@@ -49,7 +49,9 @@ $gray-light:                #818a91;
 $gray-lighter:              #eceeef;
 $gray-lightest:             #f7f7f9;
 
-//$brand-primary:             #0275d8;
+$brand-primary:             #0275d8;
```

Comment:
It might be a good idea to name these in variables above first - like $red: #f00 - and then set the the brand primary and secondary for more clarity.